x-pack/qa/sql/security/build.gradle

```diff
@@ -23,14 +23,30 @@ subprojects {
     testCompile "org.elasticsearch.plugin:x-pack-core:${version}"
   }
 
+  task changeAuditLoggingRollingPolicy {
+    def toConcatenate = files("${project(':distribution').projectDir}/src/config/log4j2.properties", "${project(':x-pack').projectDir}/plugin/core/src/main/config/log4j2.properties")
```

Comment:
We prefer specifying types explicitly instead of `def`. This one would be a `FileCollection`.